orga/app/components/analysis/analysis-per-tube-or-competence.scss

```
@@ -31,17 +36,34 @@
     width: 10%;
   }
 
-  &__tube-title {
+  &__competence-title, &__tube-title {
```

mob review : suggestion : 
```suggestion
  &__title {
```